prime-router/src/test/kotlin/azure/SendFunctionTests.kt

```diff
@@ -95,13 +95,14 @@ class SendFunctionTests {
         var nextEvent: ReportEvent? = null
         setupLogger()
         setupWorkflow()
-        every { workflowEngine.handleReportEvent(any(), any(), any()) }.answers {
-            val block = thirdArg() as
+        every { workflowEngine.handleReportEvent(any(), any()) }.answers {
+            val block = secondArg() as
```

Comment:
OK, I admit it publicly:  I find mockk to be the single most inscrutable thing in our codebase.